src/repr/src/proto/mod.rs

```diff
@@ -20,13 +20,17 @@ use std::{char::CharTryFromError, num::TryFromIntError};
 /// `Proto$T` to `$T`.
 #[derive(Debug)]
 pub enum TryFromProtoError {
+    /// A wrapped [`TryFromIntError`] due to failed integer downcast.
     TryFromIntError(TryFromIntError),
+    /// A wrapped [`TryFromIntError`] due to failed [`char`] conversion.
```

Comment:
```suggestion
    /// A wrapped [`CharTryFromError `] due to failed [`char`] conversion.
```

Is there an inconsistency in the naming: `TryFromIntError` vs `CharTryFromError`?

Reply:
Sadly yes, but the variant names are mirroring the `std` errors that they are wrapping, so would I prefer to avoid renaming them here.